Library/Formula/byteman.rb

```diff
@@ -0,0 +1,23 @@
+class Byteman < Formula
+  homepage "http://byteman.jboss.org/"
+  url "http://downloads.jboss.org/byteman/2.2.1/byteman-download-2.2.1-bin.zip"
+  sha256 "c4f8b5a89163d2ff42f90f6a90025ebd165d1e9046390f97bb5657e4fd1b85bb"
+
+  def install
+    rm_f Dir["bin/*.bat"]
+    libexec.install Dir["*"]
+  end
+
+  def caveats; <<-EOS.undent
+    The home of Byteman is:
+      #{opt_libexec}
+    You may want to add the following to your .bash_profile:
+      export BYTEMAN_HOME=#{opt_libexec}
+      export PATH=${PATH}:${BYTEMAN_HOME}/bin
+    EOS
+  end
+
+  test do
+    system "#{opt_libexec}/bin/bmcheck.sh | grep usage"
```

Comment:
Don't use `grep` here; you can use `shell_output` and `assert_match` instead or just rely on the exit code.